remarks/conversion/parsing.py

```diff
@@ -361,7 +351,7 @@ def parse_rm_file(file_path: str, dims=None) -> Tuple[Tuple[TLayers, bool], str]
 
 
 def parse_v3_to_v5(data, dims: ReMarkableDimensions, is_v3, nlayers, offset):
-    output: TLayers = {"layers": [], "highlights": [], "text": []}
+    output: TMetaData = {"layers": [], "highlights": [], "text": []}
```

Comment:
The `TMetaData` here does not have the correct type.

Reply:
Oh yeah, the types are a little bit messy still.

I even mixed using them as types and as data classes.. uhhh